bloom-housing/bloom
# chore: remove invalid HOUSING_COUNSELOR_SERVICE_URL from .env.template

# Pull Request Template

## Issue Overview

This PR addresses #2442


## Description

The original issue was that Get Assistance still showed even if it didn't have a URL to link to. It appears that this problem had already been solved. However, Sean and I found that the env.template HOUSING_COUNSELOR_SERVICE_URL link was invalid, so this PR removes that link and Get Assistance no longer shows, since it doesn't link to anything. 

## How Can This Be Tested/Reviewed?

In your public .env file, keep HOUSING_COUNSELOR_SERVICE_URL, but leave it blank. You should see that Get Assistance is no longer in the navigation bar. If you add a new link, Get Assistance should come back.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
